fulltext/backends/__ocr.py

```diff
@@ -0,0 +1,52 @@
+# sudo apt-get install tesseract-ocr
+# sudo pip3 install pytesseract
+# sudo apt-get install tesseract-ocr-[lang]
+
+from fulltext.util import which
+import pytesseract
+from PIL import Image
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+EXTENSIONS = ('jpg', 'jpeg', 'bmp', 'png', 'gif')
+
+if which('tesseract') is None:
+    LOGGER.warning('CLI tool "tesseract" is required for image files backend.')
+
+
+def read(img, **kargs):
+    lang = kargs.get('lang', 'eng')
+
+    im = Image.open(img)
+
+    degree = 0
+    try:
+        exif = im._getexif()
+    except AttributeError:
+        exif = None
+
+    if exif:
+        orientation_key = 274  # cf ExifTags
```

Comment:
I would like to see this declared as a constant (uppercase) at the top of the module.